content/apply.md

```diff
@@ -1,5 +0,0 @@
----
-type: redirect
-redirectURL: 'https://docs.google.com/forms/d/e/1FAIpQLSeirnTIywKx5w3HVLny8pfZALUYmrqDOUTroGeUDESwqmrnSA/viewform?usp=sf_link'
```

Comment:
This might break existing links we might have sent around. For example, for our last meetup, we had `https://gophers.london/apply` on the slides because it is easier to remember and apply.

As this is just a redirect, I guess we can just leave it imo.

Reply:
It doesn't. Currently in this PR apply is a folder with an index.md, which in this theme translates to /apply. 

All that changes is that /apply now lands on the speaker page instead of the google form. I think this would be better since we can edit the language of the page based on feedback, to make it less intimidating to apply. (As an example I linked the talk on talks, to show we may be able to expand this page to include more helpful/actionable advice to speakers

We may also be able to embed the form onto the page itself so everything can take places on gophers.london instead of external sites.